docs/source/glossary.rst

```diff
@@ -379,6 +379,10 @@ Glossary
 	Roles
 		Permissions :dfn:`Roles` define the operations a user is allowed to perform, and are currently an ordered list of permission levels.
 
+	Server Capability
+	Server Capabilities
+		A :dfn:`Server Capability` (not to be confused with a "Capability") expresses the capacity of a :term:`cache server` to serve a particular kind of traffic. For example, a :dfn:`Server Capability` could be created named "500GB" to be assigned to :term:`cache servers` with at least 500 GigaBytes of cache storage space. :dfn:`Server Capabilities` can also be required by :term:`Delivery Services`, which will prevent :term:`cache servers` without that :dfn:`Server Capability` from being assigned to them. It also prevents :term:`Mid-tier Cache Servers` without said :term:`Server Capability` from being selected to serve upstream requests from those :term:`Edge-tier Cache Servers` assigned to the requiring :term:`Delivery Services`.
```

Comment:
Server capabilities are not limited to expressing capacity (as the first sentence alludes to) but allows you to tag servers with any arbitrary string. For example instead of capacity you could use the server capability `ram` or `disk` for ram heavy cache servers or disk heavy cache servers.

Reply:
Well maybe I should choose a different example. A cache server that has RAM disk caches has the capacity to serve traffic that that rapidly requests newly-generated content.